pkg/network/protocols/postgres/model_linux.go

```diff
@@ -60,29 +61,27 @@ func (e *EventWrapper) Operation() Operation {
 	return e.operation
 }
 
-var (
-	regexMapper = map[Operation]*regexp.Regexp{
-		CreateTableOP: regexp.MustCompile(`CREATE TABLE (\S+)`),
-		InsertOP:      regexp.MustCompile(`INSERT INTO (\S+)`),
-		DropTableOP:   regexp.MustCompile(`DROP TABLE(\s*IF\s+EXISTS\s+)?(\S+)`),
-		UpdateOP:      regexp.MustCompile(`UPDATE (\S+)`),
-		SelectOP:      regexp.MustCompile(`SELECT .* FROM (\S+)`),
-	}
-)
-
 // extractTableName extracts the table name from the query.
 func (e *EventWrapper) extractTableName() string {
-	regex, ok := regexMapper[e.Operation()]
-	if ok {
-		if matches := regex.FindSubmatch(e.Tx.getFragment()); len(matches) > 1 {
-			return string(
-				bytes.Trim( // Remove leading and trailing quotes from the table name
-					bytes.ReplaceAll(matches[len(matches)-1], []byte{0}, []byte{}), // Remove null bytes
-					"\""),
-			)
-		}
+	fragment := string(e.Tx.getFragment())
+	// Check if the string contains "IF EXISTS",
+	// temp solution for the fact that ObfuscateSQLString does not support "IF EXISTS".
+	if strings.Contains(fragment, "IF EXISTS") {
+		fragment = strings.ReplaceAll(fragment, "IF EXISTS", "")
+	}
+
+	oq, err := obfuscate.NewObfuscator(obfuscate.Config{SQL: obfuscate.SQLConfig{
+		DBMS:            obfuscate.DBMSPostgres,
+		ObfuscationMode: obfuscate.ObfuscateAndNormalize,
```

Comment:
What is the performance impact of normalization?
Is if the "cheapest" mode we can use for the purpose?

Reply:
Actually that a good point. For now we can use only normaliziation without using the obfuscation which may make the use of it "cheaper"

Reply:
Can you back that statement with benchmarks?

Reply:
BenchmarkExtractTableName-4   	 1404390	       846.4 ns/op	     440 B/op	      18 allocs/op - obfuscation and normalisation
BenchmarkExtractTableName-4   	 1592484	       669.6 ns/op	     408 B/op	      14 allocs/op - normalize only

Reply:
move the output to the PR description